### PR TITLE
Add issue button w/ some button updates

### PR DESCRIPTION
**Pull request by mctaylors**
In this PR, I've added a "Report an issue" button and a few more button-related changes:
- Add "Report an issue" button
- Add icon for "Octobot's source code"
- Rename `AboutTitleRepository` to `ButtonOpenRepository`
- Rename `OpenEventInfoButton` to `ButtonOpenEventInfo` to be consistent with other language string names
- Rename `ColorsList.cs` to `Miscellaneous.cs`
- Add ~~public static class `Links` in `Miscellaneous.cs`~~ public const strings in `Octobot.cs` to get repository & issues links

### CODE REVIEW

File: src/Miscellaneous.cs [hunk 1]
```diff
@@ -17,3 +17,12 @@ public static class ColorsList
     public static readonly Color Black = Color.Black;
     public static readonly Color White = Color.WhiteSmoke;
 }
+
+/// <summary>
+///     Contains all links used in buttons.
+/// </summary>
+public static class Links
```
Comment (Octol1ttle) [inline src/Miscellaneous.cs:24]: Please do not put multiple classes in a single file.

**Review by Octol1ttle — CHANGES_REQUESTED** [review]

File: src/Miscellaneous.cs (hunk 1 above)
Comment (mctaylors) [reply src/Miscellaneous.cs:24]: Can I use subclasses then?

Comment (Octol1ttle) [reply src/Miscellaneous.cs:24]: That would be even worse. In general you should avoid any discrepancies between the file name and the name of the class contained within that file. Having more than one class, nested or not, makes it difficult to find these classes without the help of an IDE.

File: src/Commands/AboutCommandGroup.cs [hunk 2]
```diff
@@ -101,16 +99,24 @@ private async Task<Result> SendAboutBotAsync(IUser bot, Snowflake guildId, Cance
             .WithImageUrl("https://cdn.mctaylors.ru/octobot-banner.png")
             .Build();
 
-        var button = new ButtonComponent(
+        var repositoryButton = new ButtonComponent(
+            ButtonComponentStyle.Link,
+            Messages.ButtonOpenRepository,
+            new PartialEmoji(Name: "🌐"),
+            URL: Octobot.IssuesUrl
```
Comment (Octol1ttle) [inline src/Commands/AboutCommandGroup.cs]: wrong URL used

**Review by Octol1ttle — CHANGES_REQUESTED** [review]

**Review by Octol1ttle — APPROVED** [review]

**Pull request merged**